webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const webpack = require("webpack");
+const path = require("path");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const ExtractTextPlugin = require("extract-text-webpack-plugin");
+const package = require("./package");
+const widgetName = package.widgetName;
+
+const widgetConfig = {
+    entry: `./src/components/${widgetName}Container.ts`,
+    output: {
+        path: path.resolve(__dirname, "dist/tmp"),
+        filename: `src/com/mendix/widget/custom/dropdownfilter/${widgetName}.js`,
+        libraryTarget: "umd"
+    },
+    resolve: {
+        extensions: [ ".ts", ".js" ],
+        alias: {
+            "tests": path.resolve(__dirname, "./tests")
+        }
+    },
+    module: {
+        rules: [
+            { test: /\.ts$/, use: "ts-loader" },
+            {
+                test: /\.scss$/,
+                loader: ExtractTextPlugin.extract({
+                    fallback: "style-loader",
+                    use: "css-loader!sass-loader"
+                })
+            },
+            {
+                test: /\.gif$/,
+                use: [ {
+                    loader: "url-loader",
+                    options: { limit: 8192 }
+                } ]
+            }
+        ]
+    },
+    devtool: "source-map",
+    externals: [ "react", "react-dom", /^mxui\/|^mendix\/|^dojo\/|^dijit\// ],
+    plugins: [
+        new CopyWebpackPlugin([
+            { from: "src/**/*.xml" },
+        ], { copyUnmodified: true }),
+        new ExtractTextPlugin({ filename: `./src/com/mendix/widget/custom/dropdownfilter/ui/${widgetName}.css` }),
```

Comment:
Dont use widget name in the webpack config. 
`const packageName = package.widgetName.toLowerCase();`

Reply:
Done